test/libsolidity/SemanticTest.cpp

```diff
@@ -307,7 +299,7 @@ TestCase::TestResult SemanticTest::run(ostream& _stream, string const& _linePref
 	if (m_testCaseWantsLegacyRun)
 		result = runTest(_stream, _linePrefix, _formatted, false, false);
 
-	if ((m_testCaseWantsYulRun || m_enforceViaYul) && result == TestResult::Success)
+	if (m_testCaseWantsYulRun && result == TestResult::Success)
```

Comment:
> I still haven't made ``compileViaYul: also`` the default, though, and currently it still *requires* a ``compileViaYul`` setting

In that case maybe it would be better to behave here as if `m_enforceViaYul` was always true? Because otherwise this basically disables the enforcement and I think this makes sense only if the default is to run via yul.

Reply:
Just to clarify this - this is not needed after all because the PR initially disallowed omitting `compileViaYul` and now it made it default to `also` instead. So we don't really need enforcement - the Yul run can only be be skipped if you explicitly request it.